packages/site/index.js

```diff
@@ -1,14 +1,20 @@
-import { run_sha3_256 as sha3_wasm } from "rust-perf";
+import { run_sha3_256 as sha3_wasm, vec_allocation, u8_arr_copy } from "rust-perf";
 import { sha3_256 as sha3_js } from "js-sha3";
 
 const SNAP_ID = 'local:http://localhost:6969'
 const TEST_DATA = [
-  [5, 1000],
-  [5, 10000],
-  [5, 100000],
-  [5, 1000000],
+  [5, [1000]],
+  [5, [10000]],
+  [5, [100000]],
+  [5, [1000000]],
 ];
 
+const TEST_DATA_MEM = [
+  [5, [1000, new Uint8Array(10000).fill(1,0)]],
+  [5, [1000, new Uint8Array(10000).fill(1,0)]],
+  [5, [10000, new Uint8Array(10000).fill(1,0)]],
+  [5, [100000, new Uint8Array(10000).fill(1,0)]],
```

Comment:
I also tried with more iterations and smaller Arrays (too omit overflow).
But the result it always the same, same perfomance in snap-wasm and browser-wasm.

(5) [7, 5, 5, 4, 4]
(5) [42, 20, 17, 16, 16]
(5) [239, 161, 159, 159, 158]
(5) [2837, 1581, 1589, 1579, 1571]
The std is pretty big, because first result is always bigger then the rest. But in the end results are very stable.
<img width="430" alt="image" src="https://user-images.githubusercontent.com/24699136/194420413-f21b9bb6-2104-4d9f-af54-482b337fcf18.png">